Tasks/Gradle/gradlesonar.ts

```diff
@@ -0,0 +1,46 @@
+/// <reference path="../../definitions/vsts-task-lib.d.ts" />
+/// <reference path="../../definitions/sonarqube-common.d.ts" />
+
+import path = require('path');
+import fs = require('fs');
+
+import tl = require('vsts-task-lib/task');
+import trm = require('vsts-task-lib/toolrunner');
+
+// Lowercased file names are to lessen the likelihood of xplat issues
+import sqCommon = require('sonarqube-common/sonarqube-common');
+import {SonarQubeEndpoint} from 'sonarqube-common/sonarqube-common';
+
+// Apply arguments to enable SonarQube analysis
+export function applyEnabledSonarQubeArguments(gradleRun: trm.ToolRunner):trm.ToolRunner {
+    if (!tl.getBoolInput('sqAnalysisEnabled')) {
+        console.log("SonarQube analysis is not enabled");
+        return gradleRun;
+    }
+
+    console.log("SonarQube analysis is enabled");
```

Comment:
Even for log messages? It might be simpler and/or more readable just to remove them.